pages/test_analytics/importing_json.md.erb

```diff
@@ -189,7 +189,7 @@ end
     </tr>
   </thead>
   <tbody>
-    <% TEST_ANALYTICS_JSON['test_result']['fields'].each do |field| -%>
+    <% TEST_ANALYTICS_JSON_FIELDS_TEST_RESULT['fields'].each do |field| -%>
```

Comment:
How does this know to grab `test_result` thing? 
I reckon a more sustainable approach might just be to split the YAML file into three files, and refer to them as usual.

Reply:
It *is* three YAML files. The current method unifies them into one hash. This PR stops doing that in favour of the automatic constant assignation.

See app/models/test_analytics_json.rb  at https://github.com/buildkite/docs/pull/1688/files#diff-76488f8d80ee6eee40647959662a65050aafd40648338f56aca1059bf3235d42L1

Reply:
Oh yeah, got it. Thanks! :-) 

So I guess my question is whether it's more intuitive to unify them, or leave them as three separate files, and I'd lean towards the second (regardless of what the original config is). ;-)